src/main/plots/plots_common.py

```diff
@@ -0,0 +1,60 @@
+import os
+import logging
+
+import pandas as pd
+import matplotlib.pyplot as plt
+
+from main.util.consts import EXTENSION
+from src.main.util import consts
+from src.main.plots import consts as plot_consts
+from src.main.util.file_util import get_name_from_path, get_parent_folder_name, png_file_condition, remove_file, \
+    ct_file_condition, change_extension_to, get_file_and_parent_folder_names, get_all_file_system_items
+
+FRAGMENT_COL = consts.CODE_TRACKER_COLUMN.FRAGMENT.value
+TIMESTAMP_COL = consts.CODE_TRACKER_COLUMN.TIMESTAMP.value
+CHOSEN_TASK_COL = consts.CODE_TRACKER_COLUMN.CHOSEN_TASK.value
+TASK_STATUS_COL = consts.CODE_TRACKER_COLUMN.TASK_STATUS.value
+EVENT_TYPE_COL = consts.ACTIVITY_TRACKER_COLUMN.EVENT_TYPE.value
+EVENT_DATA_COL = consts.ACTIVITY_TRACKER_COLUMN.EVENT_DATA.value
+
+log = logging.getLogger(consts.LOGGER_NAME)
+
+
+def get_short_name(path: str):
+    folder = get_parent_folder_name(path)
+    file_name = get_name_from_path(path)
+    return os.path.join(folder, file_name[:plot_consts.SHORT_NAME_LENGTH] + '...' + file_name[-plot_consts.SHORT_NAME_LENGTH:])
+
+
+def save_plot(folder_to_save: str, data_path: str, fig: plt.figure, name_prefix: str):
```

Comment:
I think we should discuss about structure of plots folder because statistics_plots file also has __save_plot method, but it is different since the plottly library is used there